docs/development/meta-yaml.md

```diff
@@ -161,14 +169,7 @@ is the source directory.
 
 ### `build/post`
 
-Shell commands to run after building the library. These are run with `bash`, and
-there are two special environment variables defined:
-
-- `$SITEPACKAGES`: The `site-packages` directory into which the package has been
```

Comment:
I guess this `SITEPACKAGES` variable does not exist anymore?

Reply:
While we're updating this, can we add info about which directory the post script is run in? Would also be helpful to mention here that when this script is run, the `dist` directory will contain the built wheel unpacked with `python -m wheel unpack` so it's possible to manually add, delete, change, move files etc. I probably should have updated the docs here when I made these changes...

Reply:
Updated docs about `build/post` key. Maybe we can change the directory where the post script is run to make it consistent with `build/script`.

Reply:
Thanks.

> Maybe we can change the directory where the post script is run to make it consistent with build/script.

I always get confused about the directories so this might be good. But if so, we should do it as a followup.